diff --git a/src/games/Shabondama.jsx b/src/games/Shabondama.jsx
index b43ef40..6006432 100644
--- a/src/games/Shabondama.jsx
+++ b/src/games/Shabondama.jsx
@@ -17,7 +17,7 @@ export default function Shabondama() {
         <HomeChip />
         <iframe
           ref={iframeRef}
-          src="/games/shabondama_v2.html"
+          src="/games/shabondama_v3.html"
           title="しゃぼんだまポン"
           allow="autoplay; fullscreen"
           allowFullScreen
