diff --git a/src/games/DoubutsuShoot.jsx b/src/games/DoubutsuShoot.jsx
index 8054497..30daff0 100644
--- a/src/games/DoubutsuShoot.jsx
+++ b/src/games/DoubutsuShoot.jsx
@@ -17,7 +17,7 @@ export default function DoubutsuShoot() {
         <HomeChip />
         <iframe
           ref={iframeRef}
-          src="/games/shoot2.html"
+          src="/games/shoot3.html"
           title="どうぶつシューティング"
           allow="autoplay; fullscreen"
           allowFullScreen
